uiTests/build.gradle

```diff
@@ -20,9 +25,17 @@ task syncTestResults(type: Sync) {
 }
 
 test {
+  // Exclude UI tests when running `./gradlew test`, unless a `-PrunUiTests` Gradle flag is provided.
+  def runUiTests = project.hasProperty('enableUiTests')
+
   onlyIf {
-    project.hasProperty('enableUiTests')
+    runUiTests
   }
+  if (runUiTests) {
+    dependsOn ':buildPlugin'
+  }
+
+  systemProperty 'intellij.version', project.findProperty('intellijVersion')
```

Comment:
Hmm maybe sth like `project.findProperty('intellijVersion') ?: intellijVersions.current`? and then we could re-allow for `$1` param in `scripts/run-ui-tests` to be missing (most likely)